bot/interact.go

```
@@ -39,8 +39,25 @@ func interactAPIHandler(botContext cmd.BotContext) http.HandlerFunc {
 		}
 
 		switch message.Type {
+		case slack.InteractionTypeMessageAction:
+			switch message.CallbackID {
+			case "report_message":
+				dialog := generateReportMessageDialog()
+				dialog.State = slackx.LinkToMessage(message.Channel.ID, message.MessageTs) // persist the message link across submission
+				if err := botContext.Client.OpenDialog(message.TriggerID, dialog); err != nil {
+					log.Println(err.Error())
```

where is this printed? should we return the error too?